### PR TITLE
#32. Add sample for `kotlinx.datetime`

### PR DESCRIPTION
**Pull request by dector**
Used small-scoping blocks and guiding comments.

Tell me if there is something to improve here.

### CODE REVIEW

File: kotlin-jvm/src/main/kotlin/playground/_main.kt [hunk 1]
```diff
@@ -22,6 +22,7 @@ fun main() {
     playground.okio.main()
     playground.retrofit.main()
     playground.sqldelight.main()
+    playground.kotlinx.datetime.main()
```
Comment (jmfayard) [inline kotlin-jvm/src/main/kotlin/playground/_main.kt:25]: Can you keep this list ordered?
This way we avoid a merge conflict everytime two PR are merged

File: kotlin-jvm/build.gradle.kts [hunk 2]
```diff
@@ -70,6 +71,7 @@ dependencies {
     implementation(Square.retrofit2.converter.moshi)
     implementation(Square.retrofit2.retrofit)
     kapt(Square.moshi.kotlinCodegen)
+    implementation("org.jetbrains.kotlinx:kotlinx-datetime:_")
```
Comment (jmfayard) [inline kotlin-jvm/build.gradle.kts:74]: Can you keep this list ordered?
This way we avoid a merge conflict everytime two PR are merged

File: kotlin-jvm/versions.properties [hunk 3]
```diff
@@ -74,3 +74,5 @@ version.retrofit2=2.9.0
 version.sqldelight=1.4.3
 
 version.com.github.ajalt..mordant=1.2.1
+
+version.kotlinx.datetime=0.1.0
```
Comment (jmfayard) [inline kotlin-jvm/versions.properties:78]: Can you run `$ ./gradlew refreshVersions` ? 
This will re-order correctly the keys in `versions.properties`

**Review by jmfayard — APPROVED** [review]

**Comment by jmfayard** [comment]
Thanks @dector the sample is quite pedagogical.
I wish it would be possible to avoid doing `Thread.sleep(500)` to not make the execution slower, but I don't really have a good alternative :)

**Comment by dector** [comment]
> I wish it would be possible to avoid doing Thread.sleep(500) to not make the execution slower, but I don't really have a good alternative :)

Aha. There are two options:
  - Decrease timeout.
  - Try to use `kotlinx-coroutines-test` (but this will expose testing library in main source set).

**Comment by jmfayard** [comment]
Decreasing timeout is fine!

File: kotlin-jvm/src/main/kotlin/playground/_main.kt (hunk 1 above)
Comment (dector) [reply kotlin-jvm/src/main/kotlin/playground/_main.kt]: Fixed.

File: kotlin-jvm/build.gradle.kts (hunk 2 above)
Comment (dector) [reply kotlin-jvm/build.gradle.kts]: Fixed.

File: kotlin-jvm/versions.properties (hunk 3 above)
Comment (dector) [reply kotlin-jvm/versions.properties]: Fixed

**Comment by dector** [comment]
@jmfayard ammended fixes

**Comment by jmfayard** [comment]
Merged, thanks @dector !

**Pull request merged**